### DISCUSSION TITLE
Make error on Raspberrypi 4

### DISCUSSION THREAD

**Discussion by parsipal** [vlang/v#14616] [Ideas]
I tried to compile V on  Raspeberry Pi 4 (4GB), but i get an error.
This is my command list:
git clone https://github.com/vlang/v
cd v
make

This is the output with a final error:

make fresh_vc
make[1]: ingresso nella directory «/home/pi/v»
rm -rf ./vc
git clone --depth 1 --quiet --single-branch https://github.com/vlang/vc ./vc
make[1]: uscita dalla directory «/home/pi/v»
cd ./vc && git clean -xf && git pull --quiet
make fresh_tcc
make[1]: ingresso nella directory «/home/pi/v»
rm -rf ./thirdparty/tcc
git clone --depth 1 --quiet --single-branch --branch thirdparty-linux-arm https://github.com/vlang/tccbin ./thirdparty/tcc
make[2]: ingresso nella directory «/home/pi/v»
make[2]: uscita dalla directory «/home/pi/v»
make[1]: uscita dalla directory «/home/pi/v»
cd ./thirdparty/tcc && git clean -xf && git pull --quiet
cc  -std=gnu99 -w -I ./thirdparty/stdatomic/nix -o v1.exe ./vc/v.c -lm -lpthread 
/usr/bin/ld: /tmp/ccBGUyA0.o: in function `atomic_load_u64':
v.c:(.text+0xb10): undefined reference to `__atomic_load_8'
/usr/bin/ld: /tmp/ccBGUyA0.o: in function `atomic_store_u64':
v.c:(.text+0xb68): undefined reference to `__atomic_store_8'
/usr/bin/ld: /tmp/ccBGUyA0.o: in function `atomic_fetch_add_u64':
v.c:(.text+0xb9c): undefined reference to `__atomic_fetch_add_8'
/usr/bin/ld: /tmp/ccBGUyA0.o: in function `atomic_fetch_sub_u64':
v.c:(.text+0xbdc): undefined reference to `__atomic_fetch_sub_8'
collect2: error: ld returned 1 exit status
make: *** [GNUmakefile:93: all] Errore 1

I tried to follow any hint on https://github.com/vlang/v/issues/12907 issue. But nothing changed.
Could anyone help me?
Thx in adv
Parsipal

**Comment by larpon** [comment]
I could get it to compile **and** use tcc on my rPi4 B using [this gist](https://gist.github.com/Larpon/998febd5f3c33f995f0ecd561c043dcb) I made.

I'm running the script in the V project root folder.
